include/aos/sm/cni.hpp

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_SM_CNI_HPP_
+#define AOS_SM_CNI_HPP_
+
+#include "aos/common/tools/string.hpp"
+#include "aos/common/tools/uuid.hpp"
+#include "aos/common/types.hpp"
+#include "aos/sm/config.hpp"
+
+namespace aos {
```

Comment:
Recently style has been changed, looks like the next one is expected:
`namespace aos::sm::cni`

Reply:
Done